tests/test_entities.py

```diff
@@ -1,223 +1,60 @@
+# -*- encoding: utf-8 -*-
 """Tests for :mod:`nailgun.entities`."""
 from datetime import datetime
-from ddt import data, ddt, unpack
 from fauxfactory import gen_integer
 from nailgun import client, config, entities
 from nailgun.entity_mixins import EntityReadMixin, NoSuchPathError
 from sys import version_info
 from unittest import TestCase
 import mock
-# pylint:disable=too-many-public-methods
+# pylint:disable=too-many-lines
+# The size of this file is a direct reflection of the size of module
+# `nailgun.entities` and the Satellite API.
+#
+# The "cell-var-from-loop" pylint warning is ignored in several test methods.
+# The warning means this:
+#
+# > "A variable used in a closure is defined in a loop. This will result in all
+# > closures using the same value for the closed-over variable."
+#
+# Why is it OK to ignore this warning? It's OK to ignore this warning because
+# the closure being defined (typically named `_assertions()`) is *immediately*
+# called.
+
+
+def _assert(self, assertion_func):
```

Comment:
Gahhh, I failed to get this done. Sorry. I've postponed this to-do until Monday. In the mean-time, have a look at #12. :smile: